include/ignition/rendering/RenderEngine.hh

```diff
@@ -172,6 +172,15 @@ namespace ignition
       /// \return True if headless mode is enable, false otherwise.
       public: virtual bool Headless() const = 0;
 
+      /// \brief Set winID from Qt
+      /// Required in OGRE 2.2
+      /// \param[in] _winID Qt MainWindow ID
```

Comment:
Can we make the documentation less Ogre / Qt specific? Could this function be useful for other engines and / or GUI frameworks?